app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ReportsController < ApplicationController
+  layout "site"
+
+  before_action :authorize_web
+  before_action :require_user
+
+  def new
+    if create_new_report_params.present?
```

Comment:
When would the parameters not be present, and what should happen then? It looks to me like the view would just bomb because `@report` would not exist, so presumably it's not supposed to happen?